spec/classes/profile/kubernetes/destination_port_spec.rb

```diff
@@ -36,3 +35,30 @@
     end
   end
 end
+
+[
+  ['https',     30443],
+].each do |service, port|
+  describe "nebula::profile::kubernetes::destination_port::#{service}" do
```

Comment:
big oof

Reply:
Yes, I share this oof. Left as an open question of how best to deal with a now-leaky abstraction.